server/models/Availability.js

```diff
@@ -1,59 +1,63 @@
 const mongoose = require("mongoose");
 
-const availabilitySchema = new mongoose.Schema({
-  sitterId: {
-    type: mongoose.Schema.Types.ObjectId,
-    required: true,
-    ref: "User"
+function formatDate(time) {
+  const hour = time.getHours();
+  const min = time.getMinutes();
+  return new Date(0, 0, 0, hour, min, 0);
+}
+
+const availableDaySchema = new mongoose.Schema({
```

Comment:
I think we can have an `isActive` field within the `availableDaySchema` to determine whether a specific day is active within the specific schedule.

```
  isActive: {
    type: Boolean,
    required: true,
    default: false
  },
```

Reply:
Thanks @rajivtitus ,

Does active day mean what day today is ?  For instance, if today is Thursday, and Thursday is active, and tomorrow the active day will be Friday. If that is the case, a user can only set active Schedule, but not active day, since the active day automatically changes, correct?

Reply:
A day being "active" means that the sitter has some availability that day, further detailed by the `start/end` fields of the schema. It's essentially a user experience improvement: sitters don't need to enter a time for every single day, they can enable whichever days they have some available time, leaving the other ones disabled.